docs/cli/resource-deleter.md

```diff
@@ -0,0 +1,119 @@
+# Resource Deleter
+
+A package which delete
+commercetools resource like
+
+- [carts](https://docs.commercetools.com/http-api-projects-carts#delete-cart)
+- [categories](https://docs.commercetools.com/http-api-projects-categories#delete-category)
+- [channels](https://docs.commercetools.com/http-api-projects-channels#delete-channel)
+- [customers](https://docs.commercetools.com/http-api-projects-customers#delete-customer)
+- [customobjects](https://docs.commercetools.com/http-api-projects-custom-objects#delete-customobject-by-id)
+- [customergroups](https://docs.commercetools.com/http-api-projects-customerGroups#delete-customergroup)
+- [discountcodes](https://docs.commercetools.com/http-api-projects-discountCodes#delete-discountcode)
+- [inventoryentries](https://docs.commercetools.com/http-api-projects-inventory#delete-an-inventoryentry)
+- [orders](https://docs.commercetools.com/http-api-projects-orders#delete-order),
+- [payments](https://docs.commercetools.com/http-api-projects-payments#delete-payment),
+- [productdiscounts](https://docs.commercetools.com/http-api-projects-productDiscounts#delete-productdiscount)
+- [products](https://docs.commercetools.com/http-api-projects-products#delete-product)
+- [producttypes](https://docs.commercetools.com/http-api-projects-productTypes#delete-producttype)
+- [reviews](https://docs.commercetools.com/http-api-projects-reviews#delete-review)
+- [shippingmethods](https://docs.commercetools.com/http-api-projects-shippingMethods#delete-shippingmethod)
+- [states](https://docs.commercetools.com/http-api-projects-states#delete-state)
+- [taxcategories](https://docs.commercetools.com/http-api-projects-taxCategories#delete-taxcategory),
+- [types](https://docs.commercetools.com/http-api-projects-types#delete-type)
+- [zones](https://docs.commercetools.com/http-api-projects-zones#delete-zone)
+
+from the [commercetools platform](https://docs.commercetools.com/).
+
+## Configuration
+
+The constructor accepts two arguments:
+
+- A required object containing the following values:
+  - `apiConfig` (Object): `AuthMiddleware` options for authentication on the commercetools platform. (Required. See [here](https://commercetools.github.io/nodejs/sdk/api/sdkMiddlewareAuth.html#named-arguments-options))
+  - `accessToken` (String): [Access token] to be used to authenticate requests to API. Requires scope of [`manage_products`, `manage_customers`, `manage_types`]. More info on how to get the access token [here](https://docs.commercetools.com/http-api-authorization.html#authorization-flows)
+  - `resource` (Object): [resource] that need to be deleted.
+  - `predicate` (String): Query string specifying (where) predicate. More info on predicates [here](https://docs.commercetools.com/http-api.html#predicates) (Optional)
+- An optional object containing the following values:
+  - deleteHours (Number): Amount of time to delete a resource (lastModified)
```

Comment:
I see this is both documented and also an option inside the CLI but there is no functionality for it?

Reply:
more documentation, the better.